### PR TITLE
add cuav neo series gps

### PR DESCRIPTION
**Pull request by cuhome**

### CODE REVIEW

File: en/dronecan/README.md [hunk 1]
```diff
@@ -37,18 +37,19 @@ Supported hardware includes (this is not an exhaustive list):
...
-  - [Holybro DroneCAN H-RTK F9P Helical](https://holybro.com/products/dronecan-h-rtk-f9p-helical)
+- GNSS receivers for GNSS (GPS, GLONASS, BeiDou, and so on)
   - [ARK GPS](../dronecan/ark_gps.md)
   - [ARK RTK GPS](../dronecan/ark_rtk_gps.md)
   - [CubePilot Here3](https://www.cubepilot.org/#/here/here3)
-  - [Zubax GNSS](https://zubax.com/products/gnss_2)
   - [CUAV NEO v2 Pro GNSS](https://doc.cuav.net/gps/neo-series-gnss/en/neo-v2-pro.html)
-  - [CUAV NEO 3 Pro GNSS](https://doc.cuav.net/gps/neo-series-gnss/en/neo-3.html)
+  - [CUAV NEO 3 Pro GPS](../gps_compass/gps_cuav_neo_3pro.md)
+  - [CUAV NEO 3X GPS](../gps_compass/gps_cuav_neo_3x.md)
```
Comment (hamishwillee) [inline en/dronecan/README.md:46]: FYI only, added to DroneCan page (alphabetic order)

File: en/gps_compass/gps_cuav_neo_3.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+| Sensitivity                                       | Tracking and nav-167dBm<br>Cold start Hot star-148dBm<br>Reacquisition -160dBm |
+| Protocol                                          | UART+IO+I2C                                                                    |
+| Port TYpe                                         | GHR-10V-S                                                                      |
+| Supported flight controller                       | CUAV series,<br>Pixahwk series                                                 |
+| Wave filtering                                    | SAW+LNA+SAW                                                                    |
+| Anti-Electromagnetic/radio frequency interference | EMI+RFI                                                                        |
+| Upgrade firmware                                  | support                                                                        |
+| Support Flight control type                       | Ardupilot/PX4\*(PX4 is still working hard to adapt to the clock)               |
```
Comment (hamishwillee) [inline en/gps_compass/gps_cuav_neo_3.md:35]: What does this mean? "Support Flight control type " in the datasheet is something like:

- CUAV series flight control
- Pixhawk series (buzzer does not work)

Either way, what does this mean to users?

**Comment by hamishwillee** [comment]
Thanks @cuhome - I've subedited to docs standards, which mainly means:
- Images use markdown format and are shrunk
- Prettier is run on the source.
- Standard names are used for sections like "PX4 configuration".
- I also added a new "badge" to indicate the version support. This badge is a bit experimental, but I'm liking it :-)

There is an open question for you before this merges: https://github.com/PX4/PX4-user_guide/pull/2984/files#r1432287761

File: en/gps_compass/gps_cuav_neo_3.md (hunk 2 above)
Comment (cuhome) [reply en/gps_compass/gps_cuav_neo_3.md:35]: Please help me remove (buzzer does not work), the previous CUAV GPS internal buzzer is not compatible with pixhawk, but starting from neo 3, it is perfectly compatible with pixhawk standard.

**Comment by cuhome** [comment]
@hamishwillee Thank you very much.

File: en/gps_compass/gps_cuav_neo_3.md (hunk 2 above)
Comment (hamishwillee) [reply en/gps_compass/gps_cuav_neo_3.md]: Done!

**Comment by hamishwillee** [comment]
Thanks very much. Merged.

**Pull request merged**